deploy-as-code/helm/environments/unified-works-dev.yaml

```diff
@@ -139,7 +139,6 @@ cluster-configs:
         egov-edcr: "http://egov-edcr.egov:8080/"
         echallan-calculator: http://echallan-calculator.egov:8080/
         echallan-services: http://echallan-services.egov:8080/
-        inbox: "http://inbox.sanitation:8080/"

```

Comment:
@shubhang-eGov why you cange inbox?

Reply:
we shouldn't be using sanitation inbox, it was duplicate so i removed it